### PR TITLE
Remove unessesary indentation on code

### PR DESCRIPTION
**Pull request by JacobValdemar**
**Description:**

The indentation on the code is quite annoying when copying it. I haven't touched the content itself. I only changed the indentation. The `:` I added to `Upgrade the kubelet and kubectl` is to make it equal to the exactly same section further below.

[Link to issue in the docs](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#upgrade-kubelet-and-kubectl)

Before:
image

After:
image

### CODE REVIEW

**Comment by neolit123** [comment]
thanks for the change.

the preview is here:
https://deploy-preview-29748--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#upgrade-kubelet-and-kubectl

is it possible to add another new line after the "tab widget" before "Restart the kubelet:"?

**Comment by neolit123** [comment]
looks like the new line is not working, which is probably a markdown sideeffect.
i think it's safe to remove the last commit or leave it as is.

LGTM
/assign @celestehorgan
for review / approval.

**Comment by JacobValdemar** [comment]
I undid the newlines because I don't want to make people looking at this in the feature think that they should also insert two newlines :)

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -179,14 +178,11 @@ Also calling `kubeadm upgrade plan` and upgrading the CNI provider plugin is no
...
-{{< tab name="CentOS, RHEL or Fedora" >}}
-    <pre>
+{{% /tab %}}
+{{% tab name="CentOS, RHEL or Fedora" %}}
     # replace x in {{< skew currentVersion >}}.x-0 with the latest patch version
     yum install -y kubelet-{{< skew currentVersion >}}.x-0 kubectl-{{< skew currentVersion >}}.x-0 --disableexcludes=kubernetes
-    </pre>
-{{< /tab >}}
+{{% /tab %}}
 {{< /tabs >}}
```
Comment (RinkiyaKeDad) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:187]: Would it be possible to add a new line here right before the `Restart the kubelet` section begins? 

image
Looks a bit cramped currently 😅

Comment (JacobValdemar) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:187]: I totally agree! 😅 However, I already tried that in https://github.com/kubernetes/website/pull/29748/commits/cc8bda3f410cfe51f2d04060a6e1da2389b24f98, and it didn't seem to work 🤔 Do you have a different suggestion to where I could add the newline?

Comment (RinkiyaKeDad) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:187]: I think `ENTER` for new line in markdown works only once. If you want to force a line break using `<br />` might work, could you please try that?

**Comment by RinkiyaKeDad** [comment]
Looks perfect now, thanks! 

/unhold
/lgtm

**Comment by kbhawkey** [comment]
Page preview: https://deploy-preview-29748--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#upgrade-kubelet-and-kubectl-1

/kind cleanup

**Comment by kbhawkey** [comment]
Thanks @JacobValdemar .
The extra line breaks are an improvement, though we try to configure the layout of the page with CSS.
I guess the tab component requires a greater margin around the lower border.
/approve

**Pull request merged**